hooks/pre/30-upgrade.rb

```diff
@@ -74,11 +74,24 @@ def fail_and_exit(message)
   kafo.class.exit 1
 end
 
+def check_active_tasks
+  filename = "/tmp/katello_upgrade_tasks#{Time.now.to_i}"
+  filter = "state != stopped && label != Actions::Candlepin::ListenOnCandlepinEvents"
+
+  Kafo::Helpers.execute("foreman-rake foreman_tasks:export_tasks FILTER=\"#{filter}\" FORMAT=csv FILE=\"#{filename}\"")
+  task_count = `wc -l "#{filename}.csv"`.strip.split.first.to_i - 1
```

Comment:
I've put a comment in https://github.com/theforeman/foreman-tasks/pull/186/files#r64035793 when seeing here how we need to handle the suffixes.